cli/tests/testdata/workers/test.ts

```diff
@@ -454,61 +453,33 @@ Deno.test("Worker limit children permissions", async function () {
 });
 
 Deno.test("Worker limit children permissions granularly", async function () {
-  const promise = deferred();
   const worker = new Worker(
     new URL("./read_check_granular_worker.js", import.meta.url).href,
     {
       type: "module",
       deno: {
         namespace: true,
         permissions: {
-          read: [
-            new URL("./read_check_worker.js", import.meta.url),
-          ],
+          env: ["foo"],
+          hrtime: true,
+          net: ["foo", "bar:8000"],
+          ffi: [new URL("foo", import.meta.url), "bar"],
+          read: [new URL("foo", import.meta.url), "bar"],
+          run: [new URL("foo", import.meta.url), "bar", "./baz"],
+          write: [new URL("foo", import.meta.url), "bar"],
         },
       },
     },
   );
-
-  //Routes are relative to the spawned worker location
-  const routes = [
-    {
-      permission: false,
-      path: fromFileUrl(
-        new URL("read_check_granular_worker.js", import.meta.url),
-      ),
-    },
-    {
-      permission: true,
-      path: fromFileUrl(new URL("read_check_worker.js", import.meta.url)),
-    },
-  ];
-
-  let checked = 0;
-  worker.onmessage = ({ data }) => {
-    checked++;
-    assertEquals(data.hasPermission, routes[data.index].permission);
-    routes.shift();
-    if (checked === routes.length) {
-      promise.resolve();
-    }
-  };
-
-  routes.forEach(({ path }, index) =>
-    worker.postMessage({
-      index,
-      path,
-    })
-  );
-
-  await promise;
+  const promise = deferred();
+  worker.onmessage = ({ data }) => promise.resolve(data);
+  // deno-fmt-ignore
+  assertEquals(await promise, { envGlobal: "prompt", envFoo: "granted", envAbsent: "prompt", hrtime: "granted", netGlobal: "prompt", netFoo: "granted", netFoo8000: "granted", netBar: "prompt", netBar8000: "granted", ffiGlobal: "prompt", ffiFoo: "granted", ffiBar: "granted", ffiAbsent: "prompt", readGlobal: "prompt", readFoo: "granted", readBar: "granted", readAbsent: "prompt", runGlobal: "prompt", runFoo: "granted", runBar: "granted", runBaz: "granted", runAbsent: "prompt", writeGlobal: "prompt", writeFoo: "granted", writeBar: "granted", writeAbsent: "prompt" });
```

Comment:
Please factor this `assertEquals` to separate object that won't require to ignore formatting. This should format nicely with double indent